fud2/scripts/profiler.rhai

```diff
@@ -22,7 +23,7 @@ fn profiling_setup(e) {
 
     // script to process vcd and produce folded representation of flame graph
     e.var_("parse-vcd-script", "$calyx-base/tools/profiler/profiler-process.py");
-    e.rule("parse-vcd", "python3 $parse-vcd-script $in $cells profiler-out $out");
+    e.rule("parse-vcd", "python3 $parse-vcd-script $in $cells NA profiler-out $out");
```

Comment:
what does the `NA` do?

Reply:
The NA was meant to indicate that there was no source-loc mapping (i.e. the Calyx program does not have any ADL source level info to map into). But I changed this to a more elegant solution where the ADL-source-level mapping file is an optional argument to the python script.

Reply:
sounds like a good change! Thanks